### ISSUE TITLE
Increase the trade period for payment method TransferWise from 4 days to 7 days

### ISSUE THREAD

**Issue by VlVEK** [bisq-network/growth#260]
### Why

- TransferWise is a supported payment in Bisq and enables instant transfer within Europe and between its partner banks
- Outside of Europe in some countries TransferWise partners with a local bank and uses SWIFT to process the transfer
- This requires an additional period of 2-3 working days excluding weekends and public holidays

### Region

Worldwide

### Currencies

ALL

### Duration

The duration of TransferWise payment method should be increased from current 4 days to 7 days in line with the SWIFT transfer

**Comment by MwithM** [comment]
That makes sense if it's needed (@pazza83 will review more carefully) but I wanted to add that opening mediation when the trading period ends is optional.
If BTC buyer thinks that the transfer is not going to arrive before the end of trading period, he should explain why to seller and none of them shoiuld open mediation as there would be no reason for dispute.

**Comment by pazza83** [comment]
I do not think this is needed as only a minority of TransferWise payments will take 7 days. I think a most will be completed immediately to a couple of days.

I agree with @MwithM that mediation is optional and good communication between buyers and sellers will resolve most issues with when payment is expected to arrive.
